docs/make.jl

```diff
@@ -1,8 +1,8 @@
 using Pkg
 
-Pkg.develop(PackageSpec(path=joinpath(dirname(@__FILE__), "..")))
+# Pkg.develop(PackageSpec(path=joinpath(dirname(@__FILE__), "..")))
 # when first running instantiate
-Pkg.instantiate()
+# Pkg.instantiate()
```

Comment:
These can be removed for good as `build_docs.jl` does that indeed.

Reply:
Sure! Sorry, I should have removed that before doing the PR